lib/syslog.ml

```diff
@@ -54,7 +54,6 @@ let level_of_string s =
 	| "emergency"        -> Emerg
 	| "alert"            -> Alert
 	| "critical"         -> Crit
-	| "debug"            -> Debug; 
```

Comment:
It's not visible in the diff that git shows, but a bit further down in the same `match`, the `"debug"` case appears again, that is what the compiler originally complained about:
```
File "lib/syslog.ml", line 62, characters 3-10:
Warning 11: this match case is unused.
```